operator-framework-core/src/test/java/io/javaoperatorsdk/operator/processing/event/internal/TimerEventSourceTest.java

```diff
@@ -22,7 +22,7 @@
 
   public static final int INITIAL_DELAY = 50;
   public static final int PERIOD = 50;
-  public static final int TESTING_TIME_SLACK = 20;
+  public static final int TESTING_TIME_SLACK = 40;
```

Comment:
Doesn't look like it's working… 😞

Reply:
Right... maybe this type of test just isn't a good idea. Probably the Github runner is just really slow. Let me take another look.

Reply:
They also fail on my system, which, granted is not the fastest around… 😅

Reply:
I'll rewrite those tests so the timer is mocked, that's the only good solution.